scripts/jest/stack.setup.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// We want to globally mock this but jest doesn't let us do that by default
+// for a file that already exists. So we have to explicitly mock it.
+jest.mock('ReactDOMFeatureFlags', () => {
+  const flags = require.requireActual('ReactDOMFeatureFlags');
+  return Object.assign({}, flags, {
+    useFiber: false,
+  });
+});
+jest.mock('ReactNativeFeatureFlags', () => {
+  const flags = require.requireActual('ReactNativeFeatureFlags');
+  return Object.assign({}, flags, {
+    useFiber: false,
+  });
+});
+
+// Error logging varies between Fiber and Stack;
```

Comment:
How does it vary? Do we leave this after deleting Stack, or do we change something?